src/pycolmap/scene/image.cc

```diff
@@ -152,6 +152,10 @@ void BindSceneImage(py::module& m) {
       .def("reset_frame_ptr",
            &Image::ResetFramePtr,
            "Make the frame pointer a nullptr.")
+      .def("is_ref_in_frame",
+           &Image::IsRefInFrame,
+           "Check if cam_from_world needs to be composed with sensor_from_rig "
```

Comment:
nit: This comment is a bit limited. Maybe better "Check if the image was captured by the reference sensor in the rig." Same applies on the C++ side.